package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@acrolinx/sdk",
-  "version": "1.4.1",
+  "version": "1.4.2",
```

Comment:
Please checkin updated package-lock.json too.

Reply:
Thanks! I pushed in commit 11223a0b5f3c73716555a4bf217853c9ff6236e8